salt/flink/templates/flink.conf.tpl

```diff
@@ -89,19 +89,19 @@ web.port: 8081
 
 # Directory to upload completed jobs to. Add this directory to the list of
 # monitored directories of the HistoryServer as well (see below).
-#jobmanager.archive.fs.dir: hdfs:///completed-jobs/
+jobmanager.archive.fs.dir: {{ namenode }}/{{ path }}
 
 # The address under which the web-based HistoryServer listens.
-#historyserver.web.address: 0.0.0.0
+historyserver.web.address: 0.0.0.0
 
 # The port under which the web-based HistoryServer listens.
-#historyserver.web.port: 8082
+historyserver.web.port: 8082
 
 # Comma separated list of directories to monitor for completed jobs.
-#historyserver.archive.fs.dir: hdfs:///completed-jobs/
+historyserver.archive.fs.dir: {{ namenode }}/{{ path }}
```

Comment:
put a comment that the value is same as job manager archive dir.

Reply:
change done.